aws/sdk/integration-tests/ec2/tests/paginators.rs

```diff
@@ -19,7 +19,7 @@ fn stub_config() -> Config {
 /// EC2 replies with `<nextToken></nextToken>` which our XML parser parses as empty string and not "none"
 #[tokio::test]
 async fn paginators_handle_empty_tokens() {
-    let request= "Action=DescribeSpotPriceHistory&Version=2016-11-15&AvailabilityZone=eu-north-1a&InstanceType.1=g5.48xlarge&ProductDescription.1=Linux%2FUNIX";
+    let request= "Action=DescribeSpotPriceHistory&Version=2016-11-15&AvailabilityZone=eu-north-1a&DryRun=false&InstanceType.1=g5.48xlarge&MaxResults=0&ProductDescription.1=Linux%2FUNIX";
```

Comment:
Apologies again, I should have been clearer. For clients, the default should be set for responses, but not for requests. From [the spec](https://awslabs.github.io/smithy/2.0/spec/type-refinement-traits.html#default-trait):
> To allow servers to change default values if necessary, clients SHOULD NOT serialize default values unless the member is explicitly set to the default value or marked with the [default trait](https://awslabs.github.io/smithy/2.0/spec/type-refinement-traits.html#default-trait). This implies that clients SHOULD implement a kind of "presence tracking" of defaulted members so that the member is only serialized if it is explicitly set to the default value.

There shouldn't be any changes to these tests.

We need to implement presence tracking anyway for nullability support: https://github.com/awslabs/smithy-rs/issues/1767

I think you have a couple of options for moving forward:
1. Feature flag handling the default trait for clients and have it disabled for now until presence tracking is implemented. File an issue to track finishing it.
2. Implement presence tracking in this PR.